ports/fbgemm/vcpkg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "fbgemm",
+  "version-string": "2021-02-21",
```

```suggestion
  "version-date": "2021-02-21",
```
The field might be updated as `version-date`, please see the doc https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.getting-started.md#manifest-changes

---

Updated in the point in 09d912b